test/e2e/kubevirt.go

```diff
@@ -0,0 +1,628 @@
+					Name: fmt.Sprintf("worker%d", idx),
+				},
+				Spec: kubevirtv1.VirtualMachineSpec{
+					Running: pointer.Bool(true),
+					Template: &kubevirtv1.VirtualMachineInstanceTemplateSpec{
+						ObjectMeta: metav1.ObjectMeta{
+							Annotations: map[string]string{
+								"kubevirt.io/allow-pod-bridge-network-live-migration": "",
+								"k8s.ovn.org/pod-networks":                            `{"default": {"skip_ip_config": true}}`,
+							},
+							Labels: labels,
+						},
+						Spec: kubevirtv1.VirtualMachineInstanceSpec{
+							NodeSelector: map[string]string{
+								"node-role.kubernetes.io/worker": "",
+							},
+							Domain: kubevirtv1.DomainSpec{
+								Resources: kubevirtv1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceMemory: resource.MustParse("512Mi"),
+									},
+								},
+								Devices: kubevirtv1.Devices{
+									Disks: []kubevirtv1.Disk{
+										{
+											DiskDevice: kubevirtv1.DiskDevice{
+												Disk: &kubevirtv1.DiskTarget{
+													Bus: kubevirtv1.DiskBusVirtio,
+												},
+											},
+											Name: "containerdisk",
+										},
+										{
+											DiskDevice: kubevirtv1.DiskDevice{
+												Disk: &kubevirtv1.DiskTarget{
+													Bus: kubevirtv1.DiskBusVirtio,
+												},
+											},
+											Name: "cloudinitdisk",
+										},
+									},
+									Interfaces: []kubevirtv1.Interface{
+										{
+											Name: "pod",
+											InterfaceBindingMethod: kubevirtv1.InterfaceBindingMethod{
+												Bridge: &kubevirtv1.InterfaceBridge{},
+											},
+										},
+									},
+									Rng: &kubevirtv1.Rng{},
+								},
+							},
+							Networks: []kubevirtv1.Network{
+								{
+									Name: "pod",
+									NetworkSource: kubevirtv1.NetworkSource{
+										Pod: &kubevirtv1.PodNetwork{},
+									},
+								},
+							},
+							TerminationGracePeriodSeconds: pointer.Int64(5),
+							Volumes: []kubevirtv1.Volume{
+								{
+									Name: "containerdisk",
+									VolumeSource: kubevirtv1.VolumeSource{
+										ContainerDisk: &kubevirtv1.ContainerDiskSource{
+											Image: "quay.io/fedora/fedora-coreos-kubevirt:stable",
+										},
+									},
+								},
+								{
+									Name: "cloudinitdisk",
+									VolumeSource: kubevirtv1.VolumeSource{
+										CloudInitConfigDrive: &kubevirtv1.CloudInitConfigDriveSource{
+											UserData: string(ignition),
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			}, nil
+		}
+
+		composeVMs = func(numberOfVMs uint, labels map[string]string) ([]*kubevirtv1.VirtualMachine, error) {
+			vms := []*kubevirtv1.VirtualMachine{}
+			for i := uint(1); i <= numberOfVMs; i++ {
+				vm, err := composeVM(i, labels)
+				if err != nil {
+					return nil, err
+				}
+				vms = append(vms, vm)
+			}
+			return vms, nil
+		}
+
+		runTest = func(td liveMigrationTestData, vm *kubevirtv1.VirtualMachine) {
```

Comment:
this vm is in fact a kubernetes node, isn't it ? 

Do you think it makes sense to call it that then ?

Reply:
I guess it all boils down to finding a way for us to keep in mind these things are the ones subject to point to point routing.

Reply:
At ovn-kubernetes context we don't care about VMs being a kubernetes workres, we care about live migrating VMs and fullfiling connectivity requirements.